### PR TITLE
Use `execa.stdout` to get `stdout` directly

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

**Comment by gillstrom** [comment]
ThaNks MaN!

File: index.js [hunk 1]
```diff
@@ -6,12 +6,12 @@ const osxBattery = require('osx-battery');
...
 
-const win = () => execa('WMIC', ['Path', 'Win32_Battery', 'Get', 'BatteryStatus']).then(res => {
-	if (!res.stdout) {
+const win = () => execa.stdout('WMIC', ['Path', 'Win32_Battery', 'Get', 'BatteryStatus']).then(stdout => {
+	if (!stdout) {
 		return Promise.reject(new Error('No battery could be found'));
 	}
 
-	return res.stdout.indexOf('2') !== -1;
+	return stdout.indexOf('2') !== -1;
```
Comment (sindresorhus) [inline index.js:14]: You could also use `String#includes` here. It works on Node.js 4.

Comment (kevva) [reply index.js:14]: :+1:

Comment (gillstrom) [reply index.js:14]: :+1:

**Pull request merged**